README.md

```diff
@@ -1 +1,54 @@
-# MyPortfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet) 
+
+# Portfolio: setup and mobile version skeleton
+
+In this Project i have built a mobile skeleton for my microverse portfolio.
+
+![alt text](https://github.com/nedjwab/MyPortfolio/blob/mobile-skeleton/images/portfolio.PNG)
+ 
+
+## Built With 🔨
+
+- HTML
+- CSS
+- Git & Github
+
+ 
+
+### Install
+Download the zip file.
+extract the folder.
+go to the index.html file .
+
+
+### Prerequisites
+
+- IDE (  VCode For me 🔥).
+- Git.
+- Npm and node.js installed.
+
+
+### Usage
+
+- For anyone who wants to practice html5 or/and css3
+- For anyone who wants to create his own portfolio.
+
+
+## Authors
+
+👤 **Nedjwa Bouraiou**
+
+- GitHub: [@Nedjwab](https://github.com/nedjwab
```

Comment:
- [OPTIONAL] There is a missing ending parentheses `)` for the GitHub in order to be hyperlinked like LinkedIn.